packages/react-components/react-breadcrumb/docs/Spec.md

```diff
@@ -2,54 +2,479 @@
+
+### Components
+
+| Purpose                                                                | Fabric     | Northstar         | Matching? |
+| ---------------------------------------------------------------------- | ---------- | ----------------- | --------- |
+| Breadcrumb is a component that indicates the path of the current page  | Breadcrumb | Breadcrumb        | ⚠️        |
+| BreadcrumbItem an actionable item within a Breadcrumb                  |            | BreadcrumbItem    | ❌        |
+| BreadcrumbDivider divides BreadcrumbItem components within Breadcrumb  |            | BreadcrumbDivider | ❌        |
+| An BreadcrumbLink represents a anchor to be used inside the Breadcrumb |            | BreadcrumbLink    | ❌        |
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+```jsx
+const BreadcrumbV9Example = props => (
+  <Breadcrumb aria-label="breadcrumb">
+    <BreadcrumbItem href="#">
+      Home
+    </BreadcrumbItem>
+    <BreadcrumbItem onClick={() => {}}>
+      Gallery
+    </BreadcrumbItem>
+    <BreadcrumbItem href="#">
+      About
+    </BreadcrumbItem>
+  </Breadcrumb>
+  </>
+);
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+### Appearance
 
-## API
+Breadcrumb can be:
+
+- transparent (default)
+- subtle
+
+### BreadcrumbItem variants
+
+- Non-clickable element
+- Button
+- Link
+- Dropdown menu (added by a user)
+
+### Icon
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+BreadcrumbItem can have an icon with text or just icon.
+
+### Size
+
+Breadcrumb can have the following sizes: `small`, `medium` and `large`.
+
+### Tooltip
+
+Tooltips appear on collapsed menu or truncated name of item.
+
+### Dropdown menu
+
+Dropdown contains collapsed items.
+
+## API
 
 ## Structure
 
-- _**Public**_
-- _**Internal**_
-- _**DOM** - how the component will be rendered as HTML elements_
+![visual anatomy of the Breadcrumb component](./assets/breadcrumb-structure.png)
+
+### Components
+
+| Component         | Purpose                                                                 |
+| ----------------- | ----------------------------------------------------------------------- |
+| Breadcrumb        | Wrapper for the Breadcrumb component. Contains `nav` and `ol` elements. |
+| BreadcrumbDivider | Divider component                                                       |
+| BreadcrumbItem    | Contains `li` element as a wrapper. Can contain button and link         |
+| BreadcrumbMenu    | Can contain subfolders or hidden items.                                 |
+| BreadcrumbLink    | Can be used as Button or Link                                           |
```

Comment:
I called it BreadcrumbLink because it uses Link component under the hood. But our default item is Button. Not sure if it's a good name for the component

Reply:
Resolving it, cause I'll need to add more components there